### DISCUSSION TITLE
Help me,

### DISCUSSION THREAD

**Discussion by Edotamara26** [pkgxdev/pkgx#946] [Help]
For tea, up level XP ?

**Comment by codewithneeraj** [comment]
Hi @Edotamara26 

Join Tea discord https://discord.tea.xyz/ and go to roles and levels channels to get more info about xp.

I hope it's helpful.

**Reply by NahsorJr** [reply]
Just join the Tea discord, https://discord.tea.xyz/ and click on levels under general section tab and get more info about xp. 
I hyope it's helpful.

**Reply by Edotamara26** [reply]
Done, for Discord i'am join, and i'am search XP in github, how sir ?

**Comment by anhtuan111111** [comment]
You can earn points through quests in discord. If you click on #levels, there will be tasks available
Link discoed: https://discord.tea.xyz/

**Reply by Edotamara26** [reply]
Thank's for information 👍

**Comment by chris-setra** [comment]
join discord tea https://discord.tea.xyz/, and cek #levels on server

**Reply by Edotamara26** [reply]
Thank's for information 👍, and done i'am join Discord.

**Comment by nkneeraj** [comment]
> For tea, up level XP ?

As mentioned by other tea, follow - _You can earn points through quests in discord. If you click on #levels, there will be tasks available
Link discoed: https://discord.tea.xyz/_
**And stay active there you will earn 7-12 xp point for every meaningful message and don't get yourself kicked off with the bot spams.**

**Reply by Edotamara26** [reply]
Thank's for information 👍

**Comment by Cuquini95** [comment]
Its important to also Follow them On X and discord https://discord.tea.xyz/ 

Stay active, answer GitHub Questions, upload content

**Reply by Edotamara26** [reply]
Sorry, how for event ?

**Comment by tandotendo** [comment]
Just join in Discord community sir https://discord.gg/4UMwk5FW

Active in chat room and gain your XP

**Reply by Edotamara26** [reply]
For Memes or not ?

**Reply by tandotendo** [reply]
Memes, Social Engagement too

**Comment by hanriiver** [comment]
You can go discord 
command !rank
go ahead !

**Reply by Edotamara26** [reply]
Thank's for information

**Comment by gonandson** [comment]
Join tea discord https://discord.tea.xyz/ #roles and #levels

**Reply by Edotamara26** [reply]
Done sir, thank's for information.

**Comment by hanriiver** [comment]
Earn XP 

You can earn XP just from commenting, but also a few activities where the Moderators will issue XP. We strongly encourage all "help requests" to be posted to GitHub Discussions (https://github.com/teaxyz/cli/discussions), so that it is indexed by search engines. 

20XP: Create content in ⁠memes or ⁠community-content
20XP: Attend an event in ⁠office-hour or ⁠tea-time
50XP: Retweet/Engage with us on ⁠twitter
100XP: Repost a Discord Q&A on GitHub Discussions.
500XP: Answer a question on GitHub Discussions.
500XP: Create an accepted Pull Request.
2000XP: Build something with tea (release it on Hacker News, Product Hunt or dev.to).
You earn a small amount of XP for messaging (per minute), but spammers will have their XP removed.

Points will be awarded every Monday for the previous week.

**Comment by miftah770** [comment]
Need level 10
